src/ui/components/Widgets/Chart/ChartWidget.tsx

```diff
@@ -232,6 +232,35 @@ export const ChartWidget = (props: ChartWidgetProps) => {
         return changedParams;
     }, [prevSavedProps?.params, props.params, usedParamsRef, innerParamsRef, isWizardChart]);
 
+    const clearedOuterParams = React.useMemo(() => {
+        let clearedParams;
+        const propsParams = props.params;
+        const prevSavedPropsParams = prevSavedProps?.params;
+        const isEqualParamsWithPrev = prevSavedProps && isEqual(prevSavedPropsParams, propsParams);
+
+        const changedParams = !prevSavedProps || !isEqualParamsWithPrev;
```

Comment:
If it's boolean, maybe you could call it "hasParamsChanged" or something like that